xsens-tf-bridge/src/String.h

```diff
@@ -0,0 +1,94 @@
+// This is an automatically generated file.
```

Comment:
You committed all the data structures that you generated from ROS types using the `yarpidl_rosmsg` utility. To automatize this process and uniform the usage of ROS idl with Thrift idl, I would suggest that you use the `yarp_idl_to_dir` macro to automatically generate the headers of the ROS types as you already do for the Thrift files, following the "Using CMake" section in http://www.yarp.it/rostypes_tutorial_portable.html  .

http://www.yarp.it/rostypes_tutorial_portable.html